applications/FemToDemApplication/python_scripts/MainFemDem.py

```diff
@@ -242,18 +242,18 @@ def Finalize(self):
 		# Ending the problem (time integration finished)
 		self.GraphicalOutputExecuteFinalize()		
 		self.model_processes.ExecuteFinalize()
-		print(" ")
-		print("=================================================")
-		print(" - Kratos FemDem Application Calculation End   - ")
-		print("=================================================")
-		print(" ")
+		KratosMultiphysics.Logger.PrintInfo(" ")

```

Comment:
BTW: PrintInfo is not == to print. You are here basically printing a label, you should do

KratosMultiphysics.Logger.PrintInfo("FEMDEM: ", "Your real message") and instead of empty prints use "\n"-> This is a new line